lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -225,7 +225,7 @@ def self.ruby_method_runnable?(aem)
 
     def self.setup_drb_for_ruby_method
       require 'drb/timeridconv'
-      DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
+      @@global_id_conv = DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
       drb_front  = MiqAeMethodService::MiqAeServiceFront.new
       drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
```

Comment:
I pinged Alex earlier asking him to test the current PR.

On Tue, May 3, 2016 at 10:46 AM, Madhu Kanoor <notifications@github.com>
wrote:

> In lib/miq_automation_engine/engine/miq_ae_method.rb
> <https://github.com/ManageIQ/manageiq/pull/8399#discussion_r61893597>:
>
> > @@ -225,7 +225,7 @@ def self.ruby_method_runnable?(aem)
> >
> >      def self.setup_drb_for_ruby_method
> >        require 'drb/timeridconv'
> > -      DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
> > +      @@global_id_conv = DRb.install_id_conv(DRb::TimerIdConv.new(drb_cache_timeout))
> >        drb_front  = MiqAeMethodService::MiqAeServiceFront.new
> >        drb        = DRb.start_service("druby://127.0.0.1:0", drb_front)
>
> @jrafanie <https://github.com/jrafanie> @dmetzger57
> <https://github.com/dmetzger57>
> While we wait for a better solution can we check this on the performance
> team server and see if it resolves the basic issue?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/8399/files/e3c7373fe449424c5c92739cba3d560cae4b5aa0#r61893597>
>



-- 
Dennis Metzger
Engineering Supervisor - Performance & Scalability